admin-dev/themes/new-theme/js/components/router.js

```diff
@@ -43,8 +43,16 @@ const {$} = window;
  */
 export default class Router {
   constructor() {
+    if (window.prestashop && window.prestashop.customRoutes) {
```

Comment:
Could be interesting to add a part in the devdoc with all window.prestashop properties and how to use and extend them.

Reply:
https://devdocs.prestashop.com/1.7/development/components/global-components/

But it needs to be updated